packages/nimbus/src/components/combobox/combobox.recipe.tsx

```diff
@@ -0,0 +1,286 @@
+    "popover",
+    "multiSelectInput",
+    "options",
+    "optionGroup",
+    "option",
+    "optionIndicator",
+    "optionContent",
+  ],
+  // Unique class name prefix for the component
+  className: "nimbus-combobox",
+
+  // Base styles applied to all instances of the component
+  base: {
+    root: {
+      colorPalette: "primary",
+      focusRing: "outside",
+      display: "inline-flex",
+      position: "relative",
+      alignSelf: "flex-start",
+      maxWidth: "100%",
+      borderRadius: "200",
+      _hover: {
+        bg: "primary.2",
+      },
+      "&[data-disabled]": {
+        layerStyle: "disabled",
+        focusRing: "none",
+        "& input, & button": {
+          // if you set 'pointer-events: none' on an element, it does not display a cursor,
+          // so in order to display `cursor: not-allowed`, we must set pointerEvents: none on the interactive children
+          pointerEvents: "none!",
+        },
+      },
+      "& input": {
+        cursor: "text",
+        _hover: {
+          bg: "primary.2",
+        },
+        _placeholder: { opacity: 0.5 },
+      },
+      "& [data-placeholder]": {
+        opacity: 0.5,
+      },
+    },
+    value: {
+      display: "flex",
+      focusRing: "outside",
+      alignItems: "flex-start",
+      pr: "1600",
+      pl: "400",
+      py: "100",
+      borderRadius: "200",
+      color: "neutral.12",
+      maxWidth: "100%",
+      textAlign: "left",
+      whiteSpace: "nowrap",
+      overflow: "hidden",
+      textOverflow: "ellipsis",
+      boxShadow: "inset 0 0 0 var(--border-width) var(--border-color)",
+      _disabled: { pointerEvents: "none" },
+      '[data-invalid="true"] &': {
+        "--border-width": "sizes.50",
+        "--border-color": "colors.critical.7",
+        color: "critical.11",
+      },
+      '& button[slot="remove"]': {
+        _expanded: {
+          bg: "colorPalette.3",
+        },
+      },
+    },
+    buttonGroup: {
+      position: "absolute",
+      display: "inline-flex",
+      top: 0,
+      bottom: 0,
+      right: 300,
+      my: "auto",
+    },
+    popover: {
+      bg: "bg",
+      borderRadius: "200",
+      boxShadow: "5",
+    },
+    multiSelectInput: {
+      "& input": {
+        py: "300",
+        px: "400",
+        focusRing: "none",
+        borderBottom: "solid 1px",
+        borderBottomColor: "neutral.3",
+        bg: "transparent",
+        w: "100%",
+        overflow: "hidden",
+        textOverflow: "ellipsis",
+        fontSize: "350",
+        fontWeight: "400",
+        lineHeight: "400",
+        _placeholder: { opacity: 0.5 },
+      },
+    },
+    options: {
+      ...selectSlotRecipe.base?.options,
+      gap: "100",
+    },
+    optionGroup: {
+      ...selectSlotRecipe.base?.optionGroup,
+    },
+    option: {
+      ...selectSlotRecipe.base?.option,
+      whiteSpace: "wrap",
+    },
+    optionIndicator: {
+      // make sure option is aligned with the first line of text (using lh units)
```

Comment:
@FilPob @jaikamat I think I've found a way to make the checkbox top-aligned while still centering with the first line of text.  The only thing it depends on is that the first line of text is not styled by a child container that changes its' line-height.